test_cli.py

```diff
@@ -207,7 +207,17 @@ def _run_command(
 ):
     # We skip using the cli directly if force_subprocess is set to true (which forces
     # us to use subprocess even for cl commands).
-    force_subprocess = not force_subprocess and args[0] == cl
+    if args[0] == cl:
+        force_subprocess = force_subprocess if args[0] == cl else True
+        # By default, request only 4m of memory and 1m of disk so that runs are faster.
+        if len(args) > 1 and args[1] == 'run' and '--request-memory' not in args:
+            args.insert(2, '4m')
+            args.insert(2, '--request-memory')
+        if len(args) > 1 and args[1] == 'run' and '--request-disk' not in args:
+            args.insert(2, '1m')
+            args.insert(2, '--request-disk')
```

Comment:
This is kind of gross...can we have `request_memory` and `request_disk` be parameters to this function so we always append the appropriate flags rather than doing this args hacking?

Reply:
Done.